nautobot/extras/tests/test_customfields.py

```diff
@@ -1813,16 +1790,6 @@ def test_filter_multi_select(self):
             self.queryset.filter(_custom_field_data__cf9__contains="Bar"),
         )
 
-    def test_filter_null_values(self):
```

Comment:
if `null` filtering is desired, The only way i could get it to show in the UI is by using this `[("null", "None")]` instead of `[(None, "None")]